### ISSUE TITLE
Including a file containing mixins and a top-level comment produces an empty block

### ISSUE THREAD

**Issue by laurence-myers** [laurence-myers/tcat#6]
# Steps to Reproduce

main.jade
```
include ./included
p {{ 'hello' + 'world' }}
+someMixin
```

included.jade
```
// This comment will break things
mixin someMixin
    p {{ 'extra text' }}
```

The generated file will contain an empty outer block. No errors are raised.

**Comment by laurence-myers** [comment]
# Root Cause
When a comment is the first element, the `parse5` library produces a "document" object with two child nodes, a "comment" node and an "html" node.

Our HTML parsing code assumes that the first child node on the "document" is an "html" node.

The fix is to search the child nodes for the first "html" node.

**Comment by laurence-myers** [comment]
Fixed in [v0.3.0](https://github.com/laurence-myers/tcat/releases/tag/v0.3.0)
